tools/panaroo/panaroo.xml

```diff
@@ -0,0 +1,156 @@
+<tool id="panaroo" name="Panaroo" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" profile="23.02" license="MIT">
+    <description>Generate accurate pangenomes by identifying, grouping, and analyzing core and accessory genes from GFF3 files across multiple genomes.</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <edam_topics>
+        <edam_topic>topic_3945</edam_topic> <!-- "Molecular evolution " -->
+        <edam_topic>topic_0084</edam_topic> <!-- "Phylogeny" -->
+        <edam_topic>topic_3053</edam_topic> <!-- "Genetics" -->
+    </edam_topics>
+    <edam_operations>
+        <edam_operation>operation_3432</edam_operation> <!-- "Gene clustering" -->
+        <edam_operation>operation_2424</edam_operation> <!-- "Genome comparison" -->
+        <edam_operation>operation_0337</edam_operation> <!-- "Visualisation" -->
+    </edam_operations>
+    <xrefs>
+        <xref type="bio.tools">Panaroo</xref>
+    </xrefs>
+    
+    <command detect_errors="exit_code"><![CDATA[
+        ## Create the 'out' directory if it doesn't exist
+        mkdir -p out &&
+        ## Create a symbolic link to the input GFF file(s) and name it 'input.gff'
+        ln -s '$input1' input.gff
+
+        panaroo
+            -i input.gff
```

Comment:
This needs to be done, to be able to supply multiple gff3 files to panaroo as input
```suggestion
            #if $input1:
                -i ${str($input1).replace(',', ' ')}
            #end if
```